lib/transbank/sdk/patpass/patpass_by_webpay/transaction.rb

```diff
@@ -0,0 +1,73 @@
+module Transbank
+  module Patpass
+    module PatpassByWebpay
+      class Transaction
+        extend Transbank::Utils::NetHelper
+        CREATE_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+        COMMIT_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
```

Comment:
I think that this and the below lines are not related with this PR, I think that this is for a future PR.

Reply:
The PR was renamed for include the others transactions